### DISCUSSION TITLE
Where are the binaries with kraftkit?

### DISCUSSION THREAD

**Discussion by STashakkori** [unikraft/unikraft#1196] [Q&A]
When using kraftkit, which seems to be the API that the framework is moving to and wants people to use initially, it doesnt produce artifacts anywhere that I can see. Please help I just need to know where they are so I can run the unikernel on qemu manually

**Comment by STashakkori** [comment]
Found it. Its not apparent from kraftkit but from the unikraft proper side there are these instructions that got me there. Thank you for this awesome, secure product. And I hope this helps someone else someday like you guys helped me in the docs:
wget https://raw.githubusercontent.com/unikraft/app-testing/staging/scripts/generate.py -O scripts/generate.py
chmod a+x scripts/generate.py
./scripts/generate.py
 ./scripts/build/make-qemu-x86_64.sh
 Binary is in workdir/build by default will be something like: helloworld-cpp_qemu-x86_64

**Reply by craciunoiuc** [reply]
Hey, 

Nice that we found out where it was.

For the future, we are mainly using Discord for discussion, that's why the discussions on github are not that active.
